samples/ControlCatalog.iOS/ControlCatalog.iOS.csproj

```diff
@@ -177,10 +177,13 @@
       <Project>{d0a739b9-3c68-4ba6-a328-41606954b6bd}</Project>
       <Name>ControlCatalog</Name>
     </ProjectReference>
-    <PackageReference Include="SkiaSharp.HarfBuzz" Version="2.80.2-preview.33" />
```

Comment:
As far as I remember, this version was chosen on purpose, like something related to memory leaks

Reply:
Or was it done for iOS #4597

Reply:
thanks, I didn't remember. I revert it.